server/proxyUtils.js

```diff
@@ -2,7 +2,7 @@ const envVariables = require('./config/environment');
 
 const decoratePublicRequestHeaders = function () {
   return function (proxyReqOpts, srcReq) {
-    proxyReqOpts.headers["Content-Type"] = "application/json";
+    // proxyReqOpts.headers["Content-Type"] = "application/json";
```

Comment:
Why did you remove this? Always our response will be in JSON format. This can't be changed at all.

Reply:
This header overrides `reqAsBuffer` and `reqBodyEncoding` due to this content uplaod API is failing.

In the backend application, `Content-type` is not required anymore as we are passing required content-type from the fronted application and appending required headers in the backend.

For more information check link below:
https://www.npmjs.com/package/express-http-proxy#:~:text=))%3B-,parseReqBody,-The%20parseReqBody%20option